genstudio-mlr-claims-app/src/genstudiopem/web-src/src/utils/claimsValidation.ts

```diff
@@ -60,14 +73,15 @@ export const validateClaims = (experience: Experience, experienceNumber: number,
     }
 
     const result: Record<string, any[]> = {};
-    
+    const experienceFields = experience.experienceFields;
```

Comment:
nice to have would also be not to have it hardcoded to only header,pre_header, body but dynamically checking and displaying all the sections

Reply:
added